src/Entity/Course.php

```diff
@@ -237,10 +231,7 @@ public function removeContentItem(ContentItem $contentItem): self
         return $this;
     }
 
-    /**
-     * @return Collection|Report[]
-     */
-    public function getReports(): Collection
+    public function getReports(): Collection|Array
```

Comment:
This and other instances of `Array` should probably be `array` I think. Not sure if it makes a difference, but good style. Also, put a space like `Collection | array`